dask_sql/server/presto_jdbc.py

```diff
@@ -0,0 +1,150 @@
+import logging
+
+import pandas as pd
+
+from dask_sql.context import Context
+
+logger = logging.getLogger(__name__)
+
+
+def create_meta_data(c: Context):
+    """
+    Creates the schema, table and column data for prestodb JDBC driver so that data can be viewed
+    in a database tool like DBeaver. It doesn't create a catalog entry although JDBC expects one
+    as dask-sql doesn't support catalogs. For both catalogs and procedures empty placeholder
+    tables are created.
+
+    The meta-data appears in a separate schema called system_jdbc largely because the JDBC driver
+    tries to access system.jdbc and it sufficiently so shouldn't clash with other schemas.
+
+    A function is required in the /v1/statement to change system.jdbc to system_jdbc and ignore
+    order by statements from the driver (as adjust_for_presto_sql above)
+
+    :param c: Context containing created tables
+    :return:
+    """
+
+    if c is None:
+        logger.warn("Context None: jdbc meta data not created")
+        return
+    catalog = ""
+    system_schema = "system_jdbc"
+    c.create_schema(system_schema)
+
+    # if we supply a catalogue dbeaver (for instance) tries to use it
+    # but dask_sql presto interface doesn't support catalog
+    # so better for now not to supply a catalog
```

Comment:
If we leave the commented code, maybe we should format this as a `TODO` to indicate that future work can be done to enable catalogs:

```suggestion
    # TODO: add support for catalogs in presto interface
    # see https://github.com/dask-contrib/dask-sql/pull/351
```